docs/.eslintrc

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
shouldn't this be same as prettier one?

Reply:
Yes it should, this is just the result of #221 moved to the ```docs``` folder on ```master```, I'm not quite done yet.

I think I'll just remove this .eslintrc file though, as it can be handled by the root one.